src/main/java/org/sonar/scm/git/blame/FileBlamer.java

```diff
@@ -78,21 +78,22 @@ public void initialize(ObjectReader objectReader, StatefulCommit commit) {
 
   /**
    * Blame all remaining regions to the commit
+   *
+   * @param source - commit that will be used to associate blame data with remaining regions in files
    */
-  public void processResult(StatefulCommit source) {
-
+  public void saveBlameDataForFiles(StatefulCommit source) {
```

Comment:
ForCommit?

Reply:
Not sure if it makes sense to save blame data for commit? Commit already contains blame data, so we could be retrieving blame data from commit (in this case StatefulCommit) and associating this blame data with files (or with regions in files).

Reply:
For me it makes a bit of sense since we are traversing commits newest to oldest, and for each commit we determine which lines of files we need to blame with the current commit information.
suggestion sounds good